source/administration-guide/manage/admin/content-flagging.rst

```diff
@@ -0,0 +1,90 @@
+
+     - **System Administrators**: System admins receive flagged posts for all teams they are part of.
+     - **Team Administrators**: Team admins receive flagged posts for their respective teams.
+2. Under **Notification Settings**, specify who receives updates at each stage of the flagging workflow when content is flagged or reviewed:
+
+   - **Notify when content is flagged**: Reviewer(s), Author.
+   - **Notify when a reviewer is assigned**: Reviewer(s).
+   - **Notify when content is removed**: Reviewer(s), Author, Reporter.
+   - **Notify on dismissal**: Reviewer(s), Author, Reporter.
+
+   All notifications are sent via the **content-review** bot as direct messages.
+
+3. Under **Additional Settings**, configure how the flagging workflow behaves:
+
+   - **Reasons for flagging**: Define the preset categories that appear in the flagging dialog (for example: **Inappropriate content**, **Sensitive data**, **Security concern**, **Harassment or abuse**, **Spam or phishing**).
+   - **Require reporters to add comment**: Set to **True** to require users to add a short explanation when flagging a message.
+   - **Require reviewers to add comment**: Set to **True** to require reviewers to add a comment when resolving a flag.
+   - **Hide message from channel while it is being reviewed**: Set to **True** to automatically hide flagged messages from the channel until review is complete.
+
+.. tip::
+   We recommend enabling **Hide message from channel while it is being reviewed** and require comments from both reporters and reviewers to maintain transparency, accountability, and an auditable record of actions.
+
+Monitor flagged messages
+------------------------
+
+When :doc:`a user flags a message </end-user-guide/collaborate/flag-messages>`, the **content-review** bot sends a direct message to all content reviewers.
+
+Each flagged message appears as a card-formatted message that includes:
+
+- **Flagged by**: The user who reported the message.
+- **Status**: The current state of the review. All flagged content starts in **Pending** status.
+- **Reason**: The reason selected by the reporter (for example, **Sensitive data**, **Inappropriate content**).
+- **Message preview**: A snippet of the flagged message, including the author, timestamp, and original channel.
+- **Reviewer**: The user currently assigned to review the message (initially **Unassigned**).
+- **Channel**: The name of the channel where the message was originally posted.
+- **Team**: The team context for the flagged message.
+- **Comment**: Any reporter-provided context.
+- **Post ID**: The system identifier for the original message for auditing purposes.
+
+Reviewers can select **View details** to take action as follows:
+
+- Assign a **Reviewer** responsible for reviewing the flagged message.
+- **Remove message**: Permanently delete the flagged message from its original channel for all users. The status of the flagged message changes to **Removed**.
```

Comment:
It would be worth adding a warning here that this deleted the posts and all its associated contents permanently from the database and this is an irreversible operation.

The associated post data that are deleted are-

1. corresponding file infos
2. file infos associated to post's edit history
3. post's edit history
4. the files from file storage
5. post's priority data
6. post's post acknowledgements
7. post reminders
8. Scrub the post's content - message, props